pkg/controllers/customeradmin/group_controller.go

```diff
@@ -97,8 +98,20 @@ func (r *reconcileGroup) load(path string) error {
 
 func (r *reconcileGroup) pollEvent(event.GenericEvent) bool {
 	r.log.Debug("AAD Group Reconciler (poll)..")
+
+	startTime := time.Now()
+	metricLabels := prometheus.Labels{"controller": "customeradmin-group-controller"}
+
+	azureControllersInFlightGauge.With(metricLabels).Inc()
```

Comment:
What do the increment and decrement calls do?

Reply:
The Inc() and Dec() calls increment and decrement the value of the gauge respectively